File: EstCondicional/Ejercicio316.py
#Declarar Variables
puntos:int
bono:float=0
sueldoMin:float
#Datos de Entrada
puntos=int(input("Ingrese la puntuacion del docente:"))
sueldoMin=float(input("Ingrese el sueldo minimo:"))
#Proceso y Datos de Salida
if puntos>=0 and puntos<=100:
  bono=sueldoMin
elif puntos>=101 and puntos<=150:
  bono=2*sueldoMin
elif puntos>=151:
  bono=3*sueldoMin
#Datos de Salida  
print(f"El Bono que obtendra el profesor es: {bono}")

File: EstCondicional/Ejercicio316.java

import java.util.Scanner;

public class Ejercicio316 {
    static Scanner lt=new Scanner(System.in);
    public static void main(String[] args) {
        //Declarar Variables
        int puntos;
        double bono=0, sueldoMin;
        //Datos de entrada
        System.out.println("Ingrese la puntuacion del docente:");
        puntos=lt.nextInt();
        System.out.println("Ingrese el sueldo minimo:");
        sueldoMin=lt.nextDouble();
        //Proceso
        if (puntos>=0 && puntos<=100) {
            bono=sueldoMin;
        } else if( puntos>=101 && puntos<=150){
            bono=2*sueldoMin;
        }else if(puntos>=151){
            bono=3*sueldoMin;
        }
        //Datos de salida
        System.out.println("El Bono que obtendra el profesor es: "+bono);
    }
}
